content/en/docs/tasks/manage-gpus/scheduling-gpus.md

```diff
@@ -48,95 +46,24 @@ when using GPUs:
     must be equal.
   * You cannot specify GPU `requests` without specifying `limits`.
 - Containers (and Pods) do not share GPUs. There's no overcommitting of GPUs.
```

Comment:
This is not true any more either.  Nvidia supports time slicing and there are external projects for fractionalization.  Intel GPU plugin has had sharing option from start (maybe 4 years?), and  supports nowadays fractionalization.